src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -1869,14 +1933,21 @@ def aks_create(cmd, client, resource_group_name, name, ssh_key_value,  # pylint:
         resource_group_name,
         {},
         workspace_resource_id,
-        aci_subnet_name,
-        vnet_subnet_id
```

Comment:
Here, two parameters are removed by mistake, so later appgw related parameters are not aligned.